src/server/index.js

```diff
@@ -1,14 +1,51 @@
 // ««««««««« modules »»»»»»»»»
-const dotenv = require('dotenv').config();
+require('dotenv').config();
 const path = require('path');
 const express = require('express');
 const bodyParser = require('body-parser');
+const Sequelize = require('sequelize');
 const router = require('./routes.js');
 
-
 const app = express();
 const port = process.env.PORT || 8090;
 
+// ----- Postgres -----
```

Comment:
There is a database folder where all database processes are automated via Sequelize-CLI for multiple environments. Will need to discuss further.